web/src/filter.ts

```diff
@@ -189,8 +239,20 @@ function message_matches_search_term(message, operator, operand) {
     return true; // unknown operators return true (effectively ignored)
 }
 
+type Term = {
```

Comment:
I wasn't sure what to call this -- sometimes it's used on variables called "operator" which seems confusing because it has an operator attribute, sometimes it's used on things called terms (which mostly makes sense to me, as a "search term". Sometimes it's called params?

Term seems most accurate. I'm wondering if it would make sense to go through and rename all the "operator" variables which are actually terms to use the word "term" instead?

Reply:
Separated this into a separate PR since it seems a bit gnarly #28319